CDTDatastore.podspec

```diff
@@ -29,27 +29,53 @@ Pod::Spec.new do |s|
 
   s.ios.deployment_target = '6.0'
   s.osx.deployment_target = '10.8'
+
   s.requires_arc = true
 
-  s.source_files = 'Classes/**/*.{h,m}'
-  s.exclude_files = 'Classes/vendor/MYUtilities/*.{h,m}'
+  s.default_subspec = 'common'
+
+  s.subspec 'common' do |sp|
+    sp.prefix_header_contents = '#import "CollectionUtils.h"', '#import "Logging.h"', '#import "Test.h"'
+
+    sp.source_files = 'Classes/**/*.{h,m}'
+    sp.exclude_files = 'Classes/vendor/MYUtilities/*.{h,m}'
+
+    sp.ios.exclude_files = 'Classes/osx'
+    sp.osx.exclude_files = 'Classes/ios'
 
-  s.prefix_header_contents = '#import "CollectionUtils.h"', '#import "Logging.h"', '#import "Test.h"'
+    sp.frameworks = 'SystemConfiguration'
 
-  s.ios.exclude_files = 'Classes/osx'
-  s.osx.exclude_files = 'Classes/ios'
-  # s.public_header_files = 'Classes/common/CloudantSync.h'
+    sp.dependency 'CDTDatastore/no-arc'
+    sp.dependency 'CocoaLumberjack', '= 2.0.0-rc2'
 
-  s.dependency 'FMDB', '= 2.3'
-  s.dependency 'CocoaLumberjack', '= 2.0.0-rc2'
+    sp.default_subspec = 'SQLite'
 
-  s.frameworks = 'SystemConfiguration'
-  s.library = 'sqlite3', 'z'
+    sp.subspec 'SQLite' do |ssp|
+      ssp.library = 'sqlite3', 'z'
+      
+      ssp.dependency 'FMDB', '= 2.3'
+    end
+
+    sp.subspec 'SQLCipher' do |ssp|
+      ssp.library = 'z'
+
+      ssp.dependency 'FMDB/SQLCipher', '= 2.3'
+      ssp.xcconfig = { 'OTHER_CFLAGS' => '$(inherited) -DENCRYPT_DATABASE' }
+    end
```

Comment:
... I'm afraid there is a problem here. I re-checked the log generated by `pod lib lint` after reading this comment and I found the following when the subspec `CDTDatastore/SQLCipher` is tested:

```
...
            Requiring nested dependencies ()
            Creating possibility state for CDTDatastore/common/SQLCipher (= 0.14.0) (1 remaining)
              Attempting to activate CDTDatastore/common/SQLCipher (0.14.0)
              Activated CDTDatastore/common/SQLCipher at CDTDatastore/common/SQLCipher (0.14.0)
              Requiring nested dependencies (CDTDatastore/no-arc (= 0.14.0), CocoaLumberjack (= 2.0.0-rc2), FMDB/SQLCipher (= 2.3))
              Creating possibility state for CocoaLumberjack (= 2.0.0-rc2) (1 remaining)
                Attempting to activate CocoaLumberjack (2.0.0-rc2)
                Found existing spec (CocoaLumberjack (2.0.0-rc2))
                Creating possibility state for CDTDatastore/common/SQLite (= 0.14.0) (1 remaining)
                  Attempting to activate CDTDatastore/common/SQLite (0.14.0)
                  Activated CDTDatastore/common/SQLite at CDTDatastore/common/SQLite (0.14.0)
...
```

It looks like it is importing both subspecs, the one with the standard SQLite library and the one with SQLCipher. That can not be right.